pyrtlnet/numpy_inference.py

```diff
@@ -215,16 +215,18 @@ def run(self, test_batch: np.ndarray) -> tuple[np.ndarray, np.ndarray, int]:
 
         All calculations are done with NumPy and fxpmath.
 
-        :param test_batch: A batch of shape ``(batch_size, 12, 12)`` to run through the
-            NumPy inference implementation.
+        :param test_batch: A batch of shape (batch_size, 12, 12)
+            to run through the NumPy inference implementation.
 
         :returns: ``(layer0_outputs, layer1_outputs, actuals)``, where
-                  ``layer0_outputs`` is the first layer's raw tensor output, with shape
-                  ``(18, batch_size)``. ``layer1_outputs`` is the second layer's raw
-                  tensor output, with shape ``(10, batch_size)``. Note that these layer
-                  outputs are transposed compared to :func:`.run_tflite_model`.
-                  ``actuals`` is an :class:`numpy.ndarray` of predicted digits with
-                  shape ``(batch_size,)``
+                  ``layer0_outputs`` is the first layer's raw tensor output,
```

Comment:
It looks like this just rewraps this paragraph? I think the old version is easier to read since it uses the full columns allowed by `ruff format` (88 columns)